dozer-types/src/json_types.rs

```diff
@@ -33,13 +80,115 @@ pub fn field_to_json_value(field: Field) -> Value {
         Field::Decimal(n) => Value::String(n.to_string()),
         Field::Timestamp(ts) => Value::String(ts.to_rfc3339_opts(SecondsFormat::Millis, true)),
         Field::Date(n) => Value::String(n.format(DATE_FORMAT).to_string()),
-        Field::Bson(b) => Value::from(b),
+        Field::Json(b) => json_value_to_serde_json(b).unwrap(),
```

Comment:
Let's propagate the error instead of `unwrap`.

Reply:
right will do that